src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -37,7 +37,12 @@ def setup(self, builder: Builder):
         self.counter = 0
         self.output_path = self._build_output_root(builder) / "state_table.hdf"
         self.decennial_path = self._build_output_root(builder) / "decennial_census.hdf"
+
+        self.randomness = builder.randomness.get_stream(self.name)
         self.population_view = builder.population.get_view(columns=[])
+        self.response_probability_decennial = builder.lookup.build_table(
+            data=data_values.RESPONSE_PROBABILITY_DECENNIAL
+        )
```

Comment:
Will this always be a scalar?

Reply:
no! won't always be a scalar, but we don't know exactly what it will look like / haven't gotten any guidance from RT yet